Add default keymap descriptions

ryanlaws opened the pull request:
Add default descriptions for keymaps when not specified in keymap configuration:

- `map()`
- `map_expr()`

davidgranstrom commented:
@ryanlaws Thanks, looks useful! 

If you have `luacheck` and `stylua` installed locally you can format your PR by running `make format` in the root of the scnvim repo (running `make` will run the linters as a default).

To run the unit tests run `make test` in the root of the repo. The test `map_spec.lua` should be extended and updated to match the new behaviour introduced by this PR.

Please let me know if there are any issues.

ryanlaws commented:
This is awesome - I haven't worked with linting or tests in Lua before. My bad for not seeing them. I fixed the existing test, added a new one for `map_expr`, and linted it all. 

Let me know if I missed anything else. Thank you!

davidgranstrom commented:
Looks great, thanks!
